routes/apiRouter.js

```diff
@@ -126,10 +126,34 @@ router.get("/tx/:txid", function(req, res, next) {
 
 	var promises = [];
 
-	promises.push(coreApi.getRawTransactionsWithInputs([txid]));
+	let txInputLimit = (res.locals.crawlerBot) ? 3 : -1;
+
+	promises.push(coreApi.getRawTransactionsWithInputs([txid], txInputLimit));
 
 	Promise.all(promises).then(function(results) {
-		res.json(results[0].transactions[0]);
+		var outJson = results[0].transactions[0];
```

Comment:
We probably should use `const`/`let` construct instead `var` for modern JS, just like the rest of the code.

Reply:
@pointbiz Please change your patch accordingly, `git commit --amend` it and force push. Not resolved yet.

Reply:
The codebase uses a mix of var and let. I don't follow why this one line needs changing.

Reply:
I made the requested changes regarding `var` vs `const`/`let`.